scrape.py

```diff
@@ -363,7 +365,7 @@ def generate_location_steps3(num_steps):
 
 
 def generate_location_steps4(loc, num_steps):
-    meters_delta = 150
+    meters_delta = RADIUS * math.sqrt(3) - 1
```

Comment:
RADIUS * UNIT_TRIANGLE_HEIGHT * 2  makes sense to me, but I still don't understand the - 1. Are you just trying to rind down.